### ISSUE TITLE
How to set multiple reference files for fairseq-preprocess and fairseq-train command-line tools

### ISSUE THREAD

**Issue by XinzeZhang** [pytorch/fairseq#2066]
## ❓ Questions and Help

I have one-on-one Chinese to English translation training files, where the source language is Chinese and the target language is English.Such as:
- train.cn
- train.en

And I have one Chinese source file and corresponding four English files for the validation set and test set, e.g., 
- valid.cn
- valid.en0
- valid.en1
- valid.en2
- valid.en3

The question is that how to set the arguments to add the multiple reference files in the preprocess and training command-line tools?

**Comment by lematt1991** [comment]
This isn't possible, you need to have the same number of files for each language.  Either `valid.cn` and `valid.en` or `valid.cn0`, `valid.cn1`, ..., `valid.en0`, `valid.en1`, ...
